app/Http/Controllers/Frontend/AuthController.php

```diff
@@ -0,0 +1,60 @@
+<?php
+
+namespace HotelBooking\Http\Controllers\Frontend;
+
+use Request;
+use HotelBooking\Http\Controllers\Controller;
+use Auth;
+use HotelBooking\Http\Requests\Frontend\LoginRequest;
+use Session;
+use Route;
+use Response;
+
+/**
+ * Auth Controller for frontend users
+ */
+class AuthController extends FrontendBaseController
+{
+    /**
+     * Authenticate object.
+     */
+    protected $auth;
+
+    /**
+     * Controller constructor.
+     */
+    public function __construct()
+    {
+        $this->auth = Auth::user();
+        $this->middleware('user.guest', ['except' => 'getLogout']);
+    }
+
+    /**
+     * Check log in data.
+     */
+    public function postLogin(LoginRequest $request)
+    {
+        $data = $request->only('username', 'password');
+        $login = $this->auth->attempt($data);
+        if ($login) {
+            $redirectUrl = $request->input('pathname');
+            return response()->json([
+                'status' => 'success',
+                'url' => $redirectUrl
+            ]);
+        } else {
+            Session::flash('flash_error', trans('messages.login_fail'));
+            return view('layouts.frontend.partials.flash');
```

Comment:
##### If the login success
###### -> we return a json with status success( for testing), and a url which the user stand when they request login, so they can get back to the page they where they request login.
##### If the login failed
###### -> we return a flash message and a flash view to display for user. We are using ajax for login so the view will be display inpage without refresh page